### ISSUE TITLE
xn--bay-9la.de

### ISSUE THREAD

**Issue by CodingSpiderFox** [easylist/easylist#6113]
Accidentally typed `ébay.de` (with accent) instead of `ebay.de`

I was automatically redirected to xn--bay-9la.de and chromium showed a warning.

The domain should be blocked

**Comment by llacb47** [comment]
The domain is parked, it should not be blocked.

**Comment by Khrin** [comment]
I agree with @llacb47 , I don't see any reason why this domain should be blocked by EasyList.
